aspnet/migration/aspnetcoremigration.rst

```diff
@@ -0,0 +1,190 @@
+  }
+
+Configure the webroot and port in the ``hosting.json`` file.
+webroot definition moved from project.json to hosting.json
+
+WebHostBuilder API updates
+--------------------------
+
+All classes prefixed with WebApplication have been renamed to WebHost. This includes:
+
+===========================    =========================
+IWebApplicationBuilder         IWebHostBuilder
+WebApplicationBuilder          WebHostBuilder
+IWebApplication                IWebHost
+WebApplication                 WebHost
+WebApplicationOptions          WebHostOptions
+WebApplicationDefaults         WebHostDefaults
+WebApplicationService          WebHostService
+WebApplicationConfiguration    WebHostConfiguration
+===========================    =========================
+
+In the ``project.json`` file, ``web`` no longer exists as a command in the ``commands`` section. Use ``run`` or ``dotnet <dllname>`` instead  
+
+.. code-block:: JavaScript
+
+  "commands": {
+    "web": "Microsoft.AspNet.Server.Kestrel",
+    "ef": "EntityFramework.Commands"
+  },
+
+IHostingEnvironment changes 
+---------------------------
+
+All environment variables are now prefixed with the ``ASPNETCORE_`` prefix.
+
+======================  =========================    
+Old prefix              New prefix                           
+======================  =========================  
+ASPNET_WEBROOT          ASPNETCORE_WEBROOT
+ASPNET_SERVER           ASPNETCORE_SERVER
+ASPNET_APP              ASPNETCORE_APP
+ASPNET_ENVIRONMENT      ASPNETCORE_ENVIRONMENT
+ASPNET_DETAILEDERRORS   ASPNETCORE_DETAILEDERRORS
+======================  =========================  
+
+The hosting configuration keys are now consistent with the command line, environment variables, and ``hosting.json`` values. The ``Microsoft.AspNet.Hosting.json`` configuration file was renamed to ``hosting.json``.
+
+Namespace and package name changes
+---------------------------------- 
+
+All Microsoft.AspNet.* namespaces are renamed to Microsoft.AspNetCore.*. 
+The EntityFramework.* packages and namespaces are changing to Microsoft.EntityFrameworkCore.*.
+The version numbers of the above are being reset to 1.0.0-*.
+
+Working with IIS
+----------------
+
+Middleware is now setup using ``WebHostBuilder`` and no longer called in the ``Configure`` method of the ``Startup`` class.
+
+Breaking changes in the ``publish-iis`` tool: 
+
+The name of the package that contains the publish-iis tool was changed to ``Microsoft.AspNetCore.Server.IISIntegration.Tools``. This requires changing your project.json file to inlude the "Microsoft.AspNetCore.Server.IISIntegration.Tools" package instead of the "dotnet-publish-iis" package.
+
+The tool needs now to distinguish portable apps from standalone apps to be able to write the ``web.config`` file correctly depending on the application type that is being published. This required adding a new, mandatory parameter ``--framework`` that tells the tool what framework the application was published for.
```

Comment:
This seems somewhat parenthetical to the sections of `project.json` that need to change for the updated tooling. Since there are a few changes that are somewhat difficult to verbalize, it would probably be easiest for devs to consume the changes if you just show the old `dotnet-publish-iis` sections of `project. json` and then show what it looks like now. This text makes it sound like it would be easy to make one or two minor mods, but IMO it would be easier to cut-and-paste a code block right out of the doc into their code.